### PR TITLE
Logging changes varun

### PR DESCRIPTION
**Pull request by VarunSatyadevShetty**

### CODE REVIEW

File: config_util.py [hunk 1]
```diff
@@ -106,6 +106,8 @@ class ConfigurableConstants():
     resetting_time = 685  # the Exoboot typically stops updating after 731 seconds
     controller_stop_time = 710
 
+    controller_stop_time = 710
```
Comment (unghee) [inline config_util.py]: why is there two `controller_stop_time` ?

File: exoboot.py [hunk 2]
```diff
@@ -173,6 +173,14 @@ class DataContainer:
         commanded_torque: float = None
         slack: int = None
         temperature: int = None
+
+        #Varun
```
Comment (unghee) [inline exoboot.py]: nit: I would just right `controller parameters` instead of your name

File: exoboot.py [hunk 3]
```diff
@@ -312,7 +320,10 @@ def read_data(self, loop_time=None):
             self.data.toe_fsr = self.toe_fsr_detector.value
         if self.do_include_sync:
             self.data.sync = self.sync_detector.value
-
+        self.data.rise = config.RISE_FRACTION
```
Comment (unghee) [inline exoboot.py:349]: what happens if the controller is not 4 point spline?

**Review by unghee — APPROVED** [review]

File: exoboot.py (hunk 3 above)
Comment (VarunSatyadevShetty) [reply exoboot.py:349]: it will log the default value

File: config_util.py (hunk 1 above)
Comment (VarunSatyadevShetty) [reply config_util.py]: yes, it is a mistake, there should be only one controller stop time

**Pull request merged**